core/services/eth/log_broadcaster_test.go

```diff
@@ -70,13 +70,13 @@ func TestLogBroadcaster_AwaitsInitialSubscribersOnStartup(t *testing.T) {
 	sub.On("Err").Return(nil)
 
 	chSubscribe := make(chan struct{}, 10)
-	txManager.On("SubscribeToLogs", mock.Anything, mock.Anything, mock.Anything).
+	ethClient.On("SubscribeFilterLogs", mock.Anything, mock.Anything, mock.Anything).
 		Return(sub, nil).
 		Run(func(mock.Arguments) { chSubscribe <- struct{}{} })
-	txManager.On("GetLatestBlock").Return(models.Block{Number: hexutil.Uint64(blockHeight)}, nil)
-	txManager.On("GetLogs", mock.Anything).Return([]models.Log{}, nil)
+	ethClient.On("HeaderByNumber", mock.Anything, mock.Anything).Return(&types.Header{Number: big.NewInt(blockHeight)}, nil)
```

Comment:
```suggestion
	ethClient.On("HeaderByNumber", mock.Anything, nil).Return(&types.Header{Number: big.NewInt(blockHeight)}, nil)
```

It'd be nice to tighten these mock expectations down where possible.